app/services/idv/actions/verify_document_status_action.rb

```diff
@@ -88,6 +90,7 @@ def async_state
 
       def timed_out
         delete_async
+        NewRelic::Agent.notice_error(TimeoutError.new)
```

Comment:
My feeling is that this should be an event for analytics, not a new relic error.

NR is great for capturing bad exceptions or unexpected failure modes to get the stack trace, but this seems more like an event we can alert on and track that way rather than something that needs to be debuggable via the stack trace.